runtime/compiler/il/symbol/J9Symbol.cpp

```diff
@@ -182,6 +178,10 @@ J9::Symbol::searchRecognizedField(TR::Compilation * comp, TR_ResolvedMethod * ow
        {0,0,0},{0,0,0},{0,0,0},{0,0,0},{0,0,0},{0,0,0},
        {recognizedFieldName_j, 16, 52}
       };
+
+TR::Symbol::RecognizedField
```

Comment:
why did this move? some comment related to this is warranted in the commit mesage

Reply:
The static recognized field tables were scoped inside a function that uses them. I moved them outside the function so that new API implementation can access them.